src/components/dialogs/reactive-limits/reactive-capability-curve/reactive-capability-utils.js

```diff
@@ -156,11 +156,11 @@ export const calculateCurvePointsToStore = (reactiveCapabilityCurve, equipmentTo
             if (point) {
                 let pointToStore = {
                     p: point?.[P],
-                    oldP: equipmentToModify.reactiveCapabilityCurveTable?.[index]?.p ?? null,
+                    oldP: equipmentToModify?.reactiveCapabilityCurveTable?.[index]?.p ?? null,
```

Comment:
This version is not enough to safely handle equipmentToModify being null or undefined

I suggest to enhance this check by:

`const reactiveCapabilityCurveTable = equipmentToModify?.reactiveCapabilityCurveTable || [];` 
=> This avoids any issues with accessing properties on null or undefined

`const previousPoint = reactiveCapabilityCurveTable[index] || {};`
=> avoid redundancy by checking for equipmentToModify?.reactiveCapabilityCurveTable?.[index] once

(I have another remarqs about this function (you are not required to take these remarks into account) 
like using access to p, minQ, and maxQ directly, reducing redundancy by
`if (reactiveCapabilityCurve.every(({ p, minQ, maxQ }) => p == null && minQ == null && maxQ == null)) {
        return null;
    }`
also removing unnecessary else clause, replace foreach by map to iterate ...
it could be more efficient)